src/app/Project.css

```
@@ -38,9 +38,22 @@
 }
 
 .cea-overlay-card {
+  border-radius: 12px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
   pointer-events: auto;
 }
 
+.cea-map-controls-card {
+  background-color: rgb(255, 255, 255);
+
+  height: 100%;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)